getMinMax.py

```diff
@@ -0,0 +1,35 @@
+# Time Complexity : Brute Force: O(n)
+# Space Complexity : Brute Force: O(n)
+# Did this code successfully run on Leetcode : Yes
```

Comment:
why is your space complexity O(n)

Reply:
@neha79091 He is right the time complexity is O(n) only. The number of comparisons though is 2n.

Reply:
Yup we are not using any extra DS here Devasis